src/prepend.js

```diff
@@ -0,0 +1,87 @@
+import * as parents from "./parents.js";
+import {properties} from "./children.js";
+
+/**
+ * Create a new MemberExpression node by combining an object and property
+ * @param {object | string} object
+ * @param {object | string} property
+ * @param {*} [o]
+ * @param {boolean} [o.computed] Whether to use computed syntax (e.g. `foo[bar]`) or dot syntax (e.g. `foo.bar`)
+ */
+export default function prepend (object, property, o = {}) {
```

Comment:
The other big open question is regarding the name and parameter names of this function to be as clear as possible what it's doing. On one hand, I could see `prepend(node, thing_to_be_prepended)` making sense, or something like `prepend(left, right)` (which is essentially what I have here), or we could rename the function entirely to something like `combine(left, right)`

Reply:
Read the linked issue, I've responded about some of this there.

About the signature, I think the former makes most sense.

Reply:
What do you think about `prepend(node, prependee)` to make it clear 😆

Reply:
It's completely ridiculous which is a reason to like it! 🤣🤣🤣🤣

Reply:
It is so silly but also pretty obvious which node is getting prepended 🤣 
Can you think of any less ridiculous yet still clear names?

Reply:
I think you should use it 😁